source/docs/software/wpilib-tools/robot-characterization/introduction.rst

```diff
@@ -0,0 +1,62 @@
+
+``kA`` describes the voltage needed to induce a given acceleration while overcoming the robot’s inertia (and also some electromagnetic forces in the motor IIRC). Same deal as ``kV``, it’s pretty linear, and you can calculate it by accelerating to a given speed, measuring your robot’s acceleration and voltage, and subtracting the component of voltage related to static friction and velocity (which you can do because you know ``kV`` and ``kA``).
+
+With these three coefficients, and given a desired velocity and acceleration for (one side of) your drivetrain, you can use the below equation to calculate the voltage you should apply. (It also works, albeit not as well, with just velocity.) This is very useful not just for, say, following a motion profile, but also for making your drivetrain more controllable in open-loop driving because your joystick inputs will more closely match the actual robot velocity.
+
+.. math:: V = kS \cdot sgn(\dot{d}) + kV \cdot \dot{d} + kA \cdot \ddot{d}
+
+Included Characterization Tools
+-------------------------------
+
+The robot characterization toolsuite currently supports characterization for:
+
+- Drivetrains
+- Arms
+- Elevators
+
+.. note:: Many mechanisms can be characterized by simply adapting the existing code in this library.
+
+Prerequisites
+-------------
+
+To use the Robotpy Characterization Toolsuite, you must have Python (3.6 or higher) installed on your computer, as well as the standard WPILib programming toolsuite.
+
+`Python 3.7 <https://www.python.org/downloads/>`__
+
+Installing and Launching the Toolsuite
+--------------------------------------
+
+To install the Robotpy Characterization Toolsuite, open a console and enter the following command
+
+
+.. code-block:: console
+
+   pip install frc-characterization
+
+The toolsuite, and all of its dependencies, should be automatically downloaded and installed. If you are using a windows machine and the command pip is not recognized, ensure that your python scripts folder `has been added to the PATH <https://datatofish.com/add-python-to-windows-path/>`__.
+
+Once the toolsuite has been installed, launch a new drive characterization project to ensure that it works by running the following command from powershell or a terminal window.
+
+
+.. code-block:: console
+
+   robotpy-characterization drive new
+
+The new project GUI should open momentarily. To launch other characterization projects, simply replace drive with the desired characterization type.
```

Comment:
``drive`` should have code formatting, similar to the first word of this comment.